cffi_src/types.go

```diff
@@ -108,10 +110,11 @@ func (p *Timestamp) UnmarshalJSON(bytes []byte) error {
 		return fmt.Errorf("error decoding timestamp: %w", err)
 	}
 
-	*&p.Time = time.Unix(raw, 0)
+	p.Time = time.Unix(raw, 0)
 	return nil
 }
 
+// Response is the response that is sent back to the Python client.
```

Comment:
Correct but i would like to see this comments more generalized and not focused on python itself as it is the same for node and every other language where users want to use the shared libraries

Reply:
Good point! I will adjust that.